list/List.js

```diff
@@ -467,13 +467,14 @@ define([
 		//////////// Private methods ///////////////////////////////////////
 
 		/*jshint maxcomplexity:12*/
-		_refreshAriaListboxProperty: function () {
-			this.setAttribute("role", this.isAriaListbox ? "listbox" : "grid");
-			if (this.isAriaListbox) {
+		_propagateRole: function (role) {
+			if (role === "listbox") {
 				if (this.selectionMode === "none") {
 					this.selectionMode = "single";
 				}
 				// TODO: SHOULD WE REMOVE THE FOLLOWING CODE FOR OPTIMIZATION ?
```

Comment:
I think we said we finally don't want to remove this? As switching is common so maybe we should remove that comment?